src/modules/pageTable.js

```diff
@@ -119,23 +175,201 @@ class PageTable extends React.PureComponent {
             </div>
         );
     }
+
+    _onSplitterClick() {
+        const { onSplitter } = this.props;
+        onSplitter();
+    }
 }
 
 PageTable.defaultProps = {
     bleed: true,
     fontSize: 'xsmall',
+    idPrefix: 'base',
     small: true,
+    splitter: false,
 };
 
 PageTable.propTypes = {
+    bleed: PropTypes.bool,
+    className: PropTypes.string,
+    collapsed: PropTypes.bool,
+    columns: PropTypes.array.isRequired,
+    data: PropTypes.array.isRequired,
+    fontSize: PropTypes.string,
+    idPrefix: PropTypes.string,
+    onScrollColumn: PropTypes.func,
+    onSplitter: PropTypes.func,
+    rowProps: PropTypes.func,
+    sizes: PropTypes.arrayOf(PropTypes.arrayOf(PropTypes.object)),
+    small: PropTypes.bool,
+    splitter: PropTypes.bool,
+    style: PropTypes.object,
+};
+
+class PageTableContainer extends React.Component {
+    constructor(props) {
+        super(props);
+        this._onResize = this._onResize.bind(this);
```

Comment:
We need to put `this._onResize` in a debounce function and make a the handle draggable.

Reply:
Fixed.